.env.sample

```diff
@@ -0,0 +1,6 @@
+PORT=8000
+DS_API_URL=http://localhost:8002
+DS_API_TOKEN=SUPERSECRET
+DATABASE_URL=postgres://docker:docker@127.0.0.1:5400/api-dev
+OKTA_URL_ISSUER=https://example.okta.com/oauth2/default
+OKTA_CLIENT_ID=example
```

Comment:
I appreciate the inclusion of this file for any new engineer coming into the project to be able to get a running start after pulling the project down.